packages/protocol/contracts/stability/GrandaMento.sol

```diff
@@ -54,7 +70,7 @@ contract GrandaMento is
 
   struct ExchangeProposal {
     // The exchanger/proposer of the exchange proposal.
-    address exchanger;
+    address payable exchanger;
```

Comment:
Considering changing this to `proposer`, any thoughts?

Reply:
exchanger is consistent from what I have seen in other contrats I think